Banking_System/src/Transaction.java

```diff
@@ -1,27 +0,0 @@
-public abstract class Transaction {
-
-    protected double balance;
-    protected double amount;
```

Comment:
I have two questions here:

1. it's acceptable, but I want to know why you identified these data members as `protected`? knowing that we don't have any packages here.
2. what is the need of defining these data members here? I think they could be passed as parameters, also these are account related not transaction related.